src/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Leaderboard - Jose RM</title>
+  </head>
+  <body>
+    <div>
+      <div>
+        <h1>Learderboard</h1>
+      </div>
+
+      <div class="main_area">
+        <div class="records">
+          <div class="recent_score">
+            <p>Recent Score</p>
+            <button type="submit" class="refresh">Refresh</button>
+          </div>
+          <div class="scores">
+            <ul class="list_score">
+              
+            </ul>
+          </div>
+        </div>
+
+        <div>
+          <p>Add your core</p>
+          <form class="form" action="">
+            <input class="name" type="text" placeholder="Your name" required/>
+            <input class="score" type="text" placeholder="Your score" required/>
+            <button class="add_score" type="submit">Submit</button>
```

Comment:
[OPTIONAL] 👍🏽 

I guess the `required` attribute has caused a "fill in this field" error in your leaderboard form on submit. This happened as soon as i hit the submit button.

Can we fix this, if you have the time?

Here is what I mean 👇🏽 

| Your leaderboard form  |
| :---:   | 
|<img width="391" alt="Screenshot 2023-05-18 at 18 09 15" src="https://github.com/joramot/Leaderboard_JoseRM/assets/99246359/f984eba1-387b-4882-a5c8-4eec9d6e8fc3">
  |

Reply:
Thank you Favourezeugwa, I appreciate your support, what you mention is correct, add the required attribute so that I can avoid the error when submitting the form, I really invested time with my coding colleagues to solve the problem but we did not find the cause, yes I will spend time to solve the problem problem and understand the cause. Thanks again.